bin/wmagent-couchapp-init

```diff
@@ -367,6 +367,11 @@ if __name__ == "__main__":
     
     if hasattr(wmagentConfig, "Tier0Feeder"):
         installCouchApp(wmagentConfig.JobStateMachine.couchurl, wmagentConfig.Tier0Feeder.requestDBName, "T0Request")
+        centralWMStatsURL = wmagentConfig.AgentStatusWatcher.centralWMStatsURL
+        # checks whether this is test agent. if T0 agent is setting local wmstats as central it creates the db
+        if "localhost:" in centralWMStatsURL:
+            wmstatsDBName = a.split("/")[-1].strip()
```

Comment:
centralWMStatsURL.split ;-) ?

Reply:
Sorry another copy and paste error. updated